src/backend/app/db/models.py

```diff
@@ -0,0 +1,15 @@
+
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from .database import Base
+
+class Annotation(Base):
+    __tablename__="Annotation"
+    AnnotationID = Column(Integer, primary_key=True, index=True)
```

Comment:
usually for the PostgreSQL and other SQL DB people using **lower_case_with_underscores**
https://stackoverflow.com/questions/2878248/postgresql-naming-conventions

and this also matches the PEP8 style in python

Reply:
I wish this was pointed out earlier in the database proposal stage, I still think it should be addressed to the proposal and not this task as this task follows whatever the proposal introduced

Reply:
there is not really a standard I found, that being said as long as we are consistent with whatever way we choose I don't see a problem

Reply:
maybe it not a standart for SQL (never never seen field names like that though), but it still doesn't fit PEP8 :)